handler/commentHandler.go

```diff
@@ -114,6 +116,11 @@ func HandleComment(req types.IssueCommentOuter, config config.Config) {
 		feedback, err = editReviewers(prReq, command.Type, command.Value, config)
 		break
 
+	case messageConstant, messageAliasConstant:
```

Comment:
If we address the aliasing this would be just one value.

Reply:
One value here 👍